opacus/validators/batch_norm.py

```diff
@@ -11,14 +10,7 @@
 from .utils import register_module_fixer, register_module_validator
 
 
-logging.basicConfig(
-    format="%(asctime)s:%(levelname)s:%(message)s",
-    datefmt="%m/%d/%Y %H:%M:%S",
-    stream=sys.stderr,
-)
 logger = logging.getLogger(__name__)
-logger.setLevel(level=logging.INFO)
-
```

Comment:
This messes up with global logging config - I don't think it's a right strategy to configure logger while initializing a module (cc @karthikprasad )